lib/core/output_sink/base_sink.rb

```diff
@@ -9,24 +9,66 @@
 module Core
   module OutputSink
     class BaseSink
-      def ingest(_document)
-        raise 'not implemented'
+      def initialize
+        @ingested_count = 0
+        @ingested_volume = 0
+        @deleted_count = 0
       end
 
-      def ingest_multiple(_documents)
-        raise 'not implemented'
+      def ingest(document)
+        return if document.blank?
```

Comment:
I guess `empty?` would also work because the method expects a JSON object (afaik). 

#lets-remove-active_support :)

Reply:
Lol, I'm all up to remove ActiveSupport.

It's good to handle `nil` anyway cause the document that gets here is coming from the connector, and we cannot prevent the author of the connector from calling `yield nil`. I'll update this method to warn that nil/empty object is sent, I think it's a good idea.

Reply:
Ha, never seen it, thanks for sharing! I'll give it a look :)